src/Calendar.Plugin/Shared/Controls/MonthDaysView.xaml

```diff
@@ -184,185 +184,185 @@
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
 
             <controls:DayView 
 				Grid.Row="5" Grid.Column="1" 
 				DayViewSize="{Binding DayViewSize, Source={x:Reference monthsView}}" 
 				DayViewCornerRadius="{Binding DayViewCornerRadius, Source={x:Reference monthsView}}"
-				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" />
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
 
             <controls:DayView 
 				Grid.Row="5" Grid.Column="2" 
 				DayViewSize="{Binding DayViewSize, Source={x:Reference monthsView}}" 
 				DayViewCornerRadius="{Binding DayViewCornerRadius, Source={x:Reference monthsView}}"
-				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" />
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
 
             <controls:DayView 
 				Grid.Row="5" Grid.Column="3" 
 				DayViewSize="{Binding DayViewSize, Source={x:Reference monthsView}}" 
 				DayViewCornerRadius="{Binding DayViewCornerRadius, Source={x:Reference monthsView}}"
-				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" />
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
 
             <controls:DayView 
 				Grid.Row="5" Grid.Column="4" 
 				DayViewSize="{Binding DayViewSize, Source={x:Reference monthsView}}" 
 				DayViewCornerRadius="{Binding DayViewCornerRadius, Source={x:Reference monthsView}}"
-				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" />
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
 
             <controls:DayView 
 				Grid.Row="5" Grid.Column="5" 
 				DayViewSize="{Binding DayViewSize, Source={x:Reference monthsView}}" 
 				DayViewCornerRadius="{Binding DayViewCornerRadius, Source={x:Reference monthsView}}"
-				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" />
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
 
             <controls:DayView 
 				Grid.Row="5" Grid.Column="6" 
 				DayViewSize="{Binding DayViewSize, Source={x:Reference monthsView}}" 
 				DayViewCornerRadius="{Binding DayViewCornerRadius, Source={x:Reference monthsView}}"
-				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" />
+				DaysLabelStyle="{Binding DaysLabelStyle, Source={x:Reference monthsView}}" DayTapped="{Binding DayTapped, Source={x:Reference monthsView}}"/>
```

Comment:
This seems like a too much effort to be done for every change of bindable properties of DayView
Maybe we can make use of: `InitializeDayViews` method which is called right after `InitializeComponent` in the constructor of `MonthDaysView`. There we could add a code for setting bindings for each of the day views. It would require us to test it once again, but IMO it should work well. So `DayViewSize`, `DayViewCornerRadius`,  `DaysLabelStyle` and `DayTapped` could be bound there.

Reply:
I worked on this, but I ran into issues trying to set the value within the InitializeDayViews method. It seems that values passed through XAML are not available until after the constructor has finished executing (unless the BindableProperty has a default value, in which case that value is available). Therefore, the only way I could get this working was to add a case for  DayTappedCommand within OnPropertyChanged and then looping over the daysControl list manually adding the command. While I agree that setting this 42 times on MonthDaysView.xaml isn't great, I'm not sure this is better especially if we want to add DayViewSize, DayViewCornerRadius, DaysLabelStyle and DayTapped to this implementation as it would add a few hundred iterations/UI updates per page load. I'll make the changes if you still want them, I just wanted to get some feedback before diving in.

Reply:
Problem there is that you are implementing binding mechanism manually
What you need to do is implement what xaml represents actually

so for example if we want to change title view's Binding from XAML to code we would do the following:

originally in XAML:
`TextColor="{Binding DaysTitleColor, Source={x:Reference monthsView}}"`

in code:
```
daysTitleControl.BindingContext = this;

foreach (var titleLabel in daysTitleControl.Children.OfType<Label>())
        titleLabel.SetBinding(Label.TextColorProperty, nameof(DaysTitleColor));
```

so we replicate XAML by setting binding in code, instead of setting values from bindable properties and acting on their changes

Reply:
Are the most recent changes what you were looking for or should I make additional changes?